### DISCUSSION TITLE
Welcome to Tafe230106 Discussions!

### DISCUSSION THREAD

**Discussion by 001175895** [001175895/Tafe230106#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 001175895** [comment]
1.	How to calculate Monthly Repayment?

2.	What is ToString() use for?

3.	What is ToString("C") means?

**Reply by NorakSun** [reply]
1. you can calculate Monthly repayment by using Math.Pow() for example M = ( P * i ) / (1 - 1 / Math.Pow(1 + i, t * 12)).
2. ToString() is used to converts an object to its string representation so that it is suitable for display.
3. ToString("C") means to converts a number to a string representing a currency amount.

**Comment by NorakSun** [comment]
1. What is the purpose of Math.Pow()?
2. How to calculate the monthly interest ?
3.  What mortgage calculator use for ?

**Reply by 001175895** [reply]
1. The Math. Pow() method in C# is used to compute a number raised to the power of some other number.
2. Monthly interest can be calculated for example, if the annual interest rate is 3.5%; the monthly interest rate would be 0.29% (0.035/12 = 0.0033).
3. A mortgage calculator can help to add up all income sources and compare this to all monthly debt payments. It can also factor in a potential mortgage payment and other associated housing costs (property taxes, homeownership dues, etc.). One can test different loan sizes and interest rates.
